setup.py

```diff
@@ -90,19 +90,19 @@ def run(self):
     "typing_extensions>=3.7.4; python_version < '3.10'",
     # until https://github.com/python/typing/issues/865 is fixed for python3.10
     "typing_extensions==3.10.0.0; python_version >= '3.10'",
-    "fsspec>=2021.7.0",
+    "fsspec>=2021.8.1",
     "diskcache>=5.2.1",
 ]
 
 
 # Extra dependencies for remote integrations
 
-gs = ["gcsfs==2021.7.0"]
+gs = ["gcsfs==2021.8.1"]
 gdrive = ["pydrive2[fsspec]>=1.9.1"]
-s3 = ["s3fs==2021.8.0", "aiobotocore[boto3]>1.0.1"]
-azure = ["adlfs==2021.7.1", "azure-identity>=1.4.0", "knack"]
+s3 = ["s3fs==2021.8.1", "aiobotocore[boto3]>1.0.1"]
+azure = ["adlfs==2021.8.2", "azure-identity>=1.4.0", "knack"]
 oss = ["ossfs==2021.7.5"]
```

Comment:
I guess we need dependabots for our fsspec packages and for dvc, to test with the latest released fsspec version and remind us about it.
```suggestion
oss = ["ossfs==2021.8.0"]
```